pyrasterframes/src/main/python/docs/vector-data.pymd

```diff
@@ -93,7 +93,7 @@ l8_filtered = l8 \
                 .filter(st_intersects(l8.geom, st_bufferPoint(l8.paducah, lit(50000.0)))) \
                 .filter(l8.acquisition_date > '2018-02-01') \
                 .filter(l8.acquisition_date < '2018-04-01')
-l8_filtered.select('product_id', 'entity_id', 'acquisition_date', 'cloud_cover_pct').toPandas()
+l8_filtered.select('product_id', 'entity_id', 'acquisition_date', 'cloud_cover_pct')
```

Comment:
Slipped this in as I missed it in the review of #328. Want the table output to be the same as other examples.